### ISSUE TITLE
Arrows Keys "Up And Down"

### ISSUE THREAD

**Issue by on1no** [YusukeTakeuchi/Find-in-Page-with-Preview#1]
Hi
And Sorry For My Bad English

there is No Issues 
But Please Add Arrows Keys "Up And Down"

For Example in this Forum i Search for Microsoft the results is 62 matches
https://forum.xda-developers.com/windows-10-mobile/guide-win10-mobile-offline-update-t3527340
with mouse is working great See
https://i.imgur.com/n4UXEFE.gifv

But I prefer to use the arrow keys "up and down"
But it Just move up and down "without selecting" See
https://i.imgur.com/qAqxPsF.gifv

**Comment by AJS44** [comment]
Yes, Please add this ability to navigate to the next/previous instance on the web page by using the Down/Up arrow keys. That would make your excellent utility even easier to use.

Thank you!
